caikit/runtime/http_server/http_server.py

```diff
@@ -160,6 +166,12 @@ async def validation_exception_handler(_, exc: ResponseValidationError):
             self._health_check
         )
 
+        # Add runtime info endpoint
+        # self.runtime_info_servicer = RuntimeInfoServicerImpl(self.inference_service)
+        self.app.get(RUNTIME_INFO_ENDPOINT, response_class=JSONResponse)(
+            RuntimeInfoServicerImpl().GetRuntimeInfo().version_info
```

Comment:
Currently runs (when remove request and context params) but fails with error

```
obj = {'runtime_image': '1.2.3', 'caikit_version': '0.0.1', 'foo': 'bar'}

    def _signature_from_callable(obj, *,
                                 follow_wrapper_chains=True,
                                 skip_bound_arg=True,
                                 globals=None,
                                 locals=None,
                                 eval_str=False,
                                 sigcls):
    
        """Private helper function to get signature for arbitrary
        callable objects.
        """
    
        _get_signature_of = functools.partial(_signature_from_callable,
                                    follow_wrapper_chains=follow_wrapper_chains,
                                    skip_bound_arg=skip_bound_arg,
                                    globals=globals,
                                    locals=locals,
                                    sigcls=sigcls,
                                    eval_str=eval_str)
    
        if not callable(obj):
>           raise TypeError('{!r} is not a callable object'.format(obj))
E           TypeError: {'runtime_image': '1.2.3', 'caikit_version': '0.0.1', 'foo': 'bar'} is not a callable object
```
How do I match the `JSONResponse` type with the output from the method?